beacon-chain/BUILD.bazel

```diff
@@ -44,6 +45,7 @@ container_push(
     registry = "gcr.io",
     repository = "prysmaticlabs/prysm/beacon-chain",
     tag = "latest",
```

Comment:
Should this be `tags = ["manual","latest"]`?

Reply:
Right now they are:
```
tag = "latest",
tags = ["manual"],
```
should it just be?
```
tags = ["manual","latest"]
```

Reply:
Nevermind, what you have looks correct. [tag](https://github.com/bazelbuild/rules_docker#container_pull-1) is specifc to `rules_docker`, whereas [tags](https://docs.bazel.build/versions/master/be/common-definitions.html#common-attributes) applies to all rules. Would be nice if attributes were namespaced with prefixes.